src/content/pages/attend/volunteer.mdx

```diff
@@ -55,7 +64,7 @@ PyCon AU operates under a clear [Code of Conduct](/safety) and we ask that all v
 
 If you are interested, please use the form below to send us your details, the times that you are available, and the skills you have to offer.
 
-Volunteer applications close midnight Melbourne time at the end of **Monday 07 October, 2024**.
+Volunteer applications close 5PM AEST on **Wednesday 18 September, 2024**.
```

Comment:
Might be nice to keep the old time here with ~~strikethrough~~ for transparency and put in something like "Update: due to the record number of volunteer applications we are closing volunteer applications earlier than originally planned"